serving/src/main/java/ai/djl/serving/ModelServer.java

```diff
@@ -348,8 +348,7 @@ private void initModelStore() throws IOException, BadWorkflowException {
         String overrideEngineModel = Utils.getEnvOrSystemProperty("LOAD_WITH_ENGINE");
```

Comment:
Why we need this?

Does the following work?

```
export SERVING_LOAD_MODELS=test::Python:/opt/ml/MY_MODEL;test2=...
```

Reply:
This doesn't work

```
SERVING_LOAD_MODELS=test::MPI:/opt/ml/model; OPTION_TENSOR_PARALLEL_DEGREE=1 OPTION_MODEL_ID=openlm-research/open_llama_7b_v2 OPTION_TASK=text-generation OPTION_ROLLING_BATCH=auto OPTION_DTYPE=fp16 djl-serving
```

Reply:
this worked
```
SERVING_LOAD_MODELS=test::MPI=/opt/ml/model OPTION_TENSOR_PARALLEL_DEGREE=4 OPTION_MODEL_ID=openlm-research/open_llama_7b_v2 OPTION_TASK=text-genation OPTION_ROLLING_BATCH=auto OPTION_DTYPE=fp16 djl-serving
```